src/edn_query_language/core.cljc

```diff
@@ -31,7 +30,7 @@
 
      ::gen-ident-value
      (fn gen-ident-value [_]
-       (gen/frequency [[15 gen/simple-type-printable]
+       (gen/frequency [[15 (gen/simple-type-printable)]
```

Comment:
this shouldn't have the list around, since `gen/simple-type-printable` is a generator already.

Reply:
This is calling into `clojure.spec.gen.alpha`, not test.check's generator namespace. Everything in the spec.gen namespace is a function. 
```clojure
(require '[clojure.spec.gen.alpha :as gen])
=> nil
(gen/generate (gen/simple-type-printable))
=> -0.90850830078125
(gen/generate gen/simple-type-printable)
java.lang.NullPointerException: 
```

Reply:
ah, you right, thanks for the clarification